### ISSUE TITLE
2024-11-27, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#3]
# Today's Arxiv Papers

(1) [Adaptive Deployment of Untrusted LLMs Reduces Distributed Threats](http://arxiv.org/abs/2411.17693v1), 2024-11-26

(2) [Low-Bit Quantization Favors Undertrained LLMs: Scaling Laws for Quantized LLMs with 100T Training Tokens](http://arxiv.org/abs/2411.17691v1), 2024-11-26

(3) [Rethinking Token Reduction in MLLMs: Towards a Unified Paradigm for Training-Free Acceleration](http://arxiv.org/abs/2411.17686v1), 2024-11-26

(4) [Enhancing Character-Level Understanding in LLMs through Token Internal Structure Learning](http://arxiv.org/abs/2411.17679v1), 2024-11-26

(5) [Push the Limit of Multi-modal Emotion Recognition by Prompting LLMs with Receptive-Field-Aware Attention Weighting](http://arxiv.org/abs/2411.17674v1), 2024-11-26

(6) [SketchAgent: Language-Driven Sequential Sketch Generation](http://arxiv.org/abs/2411.17673v1), 2024-11-26

(7) [Synthetic Data Generation with LLM for Improved Depression Prediction](http://arxiv.org/abs/2411.17672v1), 2024-11-26

(8) [Toward High-Performance LLM Serving: A Simulation-Based Approach for Identifying Optimal Parallelism](http://arxiv.org/abs/2411.17651v1), 2024-11-26

(9) [On Limitations of LLM as Annotator for Low Resource Languages](http://arxiv.org/abs/2411.17637v1), 2024-11-26

(10) [MALMM: Multi-Agent Large Language Models for Zero-Shot Robotics Manipulation](http://arxiv.org/abs/2411.17636v1), 2024-11-26

(11) [Data-driven development of cycle prediction models for lithium metal batteries using multi modal mining](http://arxiv.org/abs/2411.17625v1), 2024-11-26

(12) [Scaling Speech-Text Pre-training with Synthetic Interleaved Data](http://arxiv.org/abs/2411.17607v1), 2024-11-26

(13) [HyperSeg: Towards Universal Visual Segmentation with Large Language Model](http://arxiv.org/abs/2411.17606v1), 2024-11-26

(14) [Making History Readable](http://arxiv.org/abs/2411.17600v1), 2024-11-26

(15) [Agentic AI for Improving Precision in Identifying Contributions to Sustainable Development Goals](http://arxiv.org/abs/2411.17598v1), 2024-11-26

(16) [Can artificial intelligence predict clinical trial outcomes?](http://arxiv.org/abs/2411.17595v1), 2024-11-26

(17) [RTL-Breaker: Assessing the Security of LLMs against Backdoor Attacks on HDL Code Generation](http://arxiv.org/abs/2411.17569v1), 2024-11-26

(18) [Natural Language Understanding and Inference with MLLM in Visual Question Answering: A Survey](http://arxiv.org/abs/2411.17558v1), 2024-11-26

(19) [Pushing the Limits of Large Language Model Quantization via the Linearity Theorem](http://arxiv.org/abs/2411.17525v1), 2024-11-26

(20) [Inference Scaling $\scriptsize\mathtt{F}$Laws: The Limits of LLM Resampling with Imperfect Verifiers](http://arxiv.org/abs/2411.17501v1), 2024-11-26

(21) [SoK: Decentralized AI (DeAI)](http://arxiv.org/abs/2411.17461v1), 2024-11-26

(22) [PEFTGuard: Detecting Backdoor Attacks Against Parameter-Efficient Fine-Tuning](http://arxiv.org/abs/2411.17453v1), 2024-11-26

(23) ["Stupid robot, I want to speak to a human!" User Frustration Detection in Task-Oriented Dialog Systems](http://arxiv.org/abs/2411.17437v1), 2024-11-26

(24) [BPP-Search: Enhancing Tree of Thought Reasoning for Mathematical Modeling Problem Solving](http://arxiv.org/abs/2411.17404v1), 2024-11-26

(25) [One Mind, Many Tongues: A Deep Dive into Language-Agnostic Knowledge Neurons in Large Language Models](http://arxiv.org/abs/2411.17401v1), 2024-11-26

(26) [Can LLMs be Good Graph Judger for Knowledge Graph Construction?](http://arxiv.org/abs/2411.17388v1), 2024-11-26

(27) [The Extractive-Abstractive Spectrum: Uncovering Verifiability Trade-offs in LLM Generations](http://arxiv.org/abs/2411.17375v1), 2024-11-26

(28) [Different Bias Under Different Criteria: Assessing Bias in LLMs with a Fact-Based Approach](http://arxiv.org/abs/2411.17338v1), 2024-11-26

(29) [MotionLLaMA: A Unified Framework for Motion Synthesis and Comprehension](http://arxiv.org/abs/2411.17335v1), 2024-11-26

(30) [InsightEdit: Towards Better Instruction Following for Image Editing](http://arxiv.org/abs/2411.17323v1), 2024-11-26

(31) [PIM-AI: A Novel Architecture for High-Efficiency LLM Inference](http://arxiv.org/abs/2411.17309v1), 2024-11-26

(32) [Meaningless is better: hashing bias-inducing words in LLM prompts improves performance in logical reasoning and statistical learning](http://arxiv.org/abs/2411.17304v1), 2024-11-26

(33) [ER2Score: LLM-based Explainable and Customizable Metric for Assessing Radiology Reports with Reward-Control Loss](http://arxiv.org/abs/2411.17301v1), 2024-11-26

(34) [Using Large Language Models for Expert Prior Elicitation in Predictive Modelling](http://arxiv.org/abs/2411.17284v1), 2024-11-26

(35) [CleanVul: Automatic Function-Level Vulnerability Detection in Code Commits Using LLM Heuristics](http://arxiv.org/abs/2411.17274v1), 2024-11-26

(36) [A Topic-level Self-Correctional Approach to Mitigate Hallucinations in MLLMs](http://arxiv.org/abs/2411.17265v1), 2024-11-26

(37) [HEIE: MLLM-Based Hierarchical Explainable AIGC Image Implausibility Evaluator](http://arxiv.org/abs/2411.17261v1), 2024-11-26

(38) [APT: Architectural Planning and Text-to-Blueprint Construction Using Large Language Models for Open-World Agents](http://arxiv.org/abs/2411.17255v1), 2024-11-26

(39) [Grounding-IQA: Multimodal Language Grounding Model for Image Quality Assessment](http://arxiv.org/abs/2411.17237v1), 2024-11-26

(40) [Fault Localization from the Semantic Code Search Perspective](http://arxiv.org/abs/2411.17230v1), 2024-11-26

(41) [Strategic Prompting for Conversational Tasks: A Comparative Analysis of Large Language Models Across Diverse Conversational Tasks](http://arxiv.org/abs/2411.17204v1), 2024-11-26

(42) [Exploring the Impact of Anthropomorphism in Role-Playing AI Chatbots on Media Dependency: A Case Study of Xuanhe AI](http://arxiv.org/abs/2411.17157v1), 2024-11-26

(43) [LLM-Based Offline Learning for Embodied Agents via Consistency-Guided Reward Ensemble](http://arxiv.org/abs/2411.17135v1), 2024-11-26

(44) [TechCoach: Towards Technical Keypoint-Aware Descriptive Action Coaching](http://arxiv.org/abs/2411.17130v1), 2024-11-26

(45) [DOGE: Towards Versatile Visual Document Grounding and Referring](http://arxiv.org/abs/2411.17125v1), 2024-11-26

(46) [Advancing Content Moderation: Evaluating Large Language Models for Detecting Sensitive Content Across Text, Images, and Videos](http://arxiv.org/abs/2411.17123v1), 2024-11-26

(47) [Star Attention: Efficient LLM Inference over Long Sequences](http://arxiv.org/abs/2411.17116v1), 2024-11-26

(48) [TabulaX: Leveraging Large Language Models for Multi-Class Table Transformations](http://arxiv.org/abs/2411.17110v1), 2024-11-26

(49) [Scholar Name Disambiguation with Search-enhanced LLM Across Language](http://arxiv.org/abs/2411.17102v1), 2024-11-26

(50) [Efficient LLM Inference with I/O-Aware Partial KV Cache Recomputation](http://arxiv.org/abs/2411.17089v1), 2024-11-26

(51) [Creative Agents: Simulating the Systems Model of Creativity with Generative Agents](http://arxiv.org/abs/2411.17065v1), 2024-11-26

(52) [ThreatModeling-LLM: Automating Threat Modeling using Large Language Models for Banking System](http://arxiv.org/abs/2411.17058v1), 2024-11-26

(53) [Redefining Crowdsourced Test Report Prioritization: An Innovative Approach with Large Language Model](http://arxiv.org/abs/2411.17045v1), 2024-11-26

(54) [TED-VITON: Transformer-Empowered Diffusion Models for Virtual Try-On](http://arxiv.org/abs/2411.17017v1), 2024-11-26

(55) [Trinity: Synchronizing Verbal, Nonverbal, and Visual Channels to Support Academic Oral Presentation Delivery](http://arxiv.org/abs/2411.17015v1), 2024-11-26
